dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java

```diff
@@ -204,6 +204,14 @@ public void saveSchemesForStruct(final Structure struc, final List<WorkflowSchem
 		}
 	}
 
+	public void saveSchemeIdsForStruct(final ContentType contentType, List<String> schemesIds) throws DotDataException {
+		try {
+			workFlowFactory.saveSchemeIdsForStruct(contentType.inode(), schemesIds);
+		} catch(DotDataException e){
+			throw e;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add logic to this catch clause or eliminate it and rethrow the exception automatically. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2737)

Reply:
add more logging!.